### PR TITLE
[merge after v1] Dummy data (close #885)

### LINKED ISSUE #885: Add dummy data for generated tests
Currently the generated tests don't work until the args are updated as we use empty strings, this should be updated with some dummy data

### PR DESCRIPTION
**Pull request by Jakeeyturner**
Signed-off-by: Jake Turner <jaketurner25@live.com>

### CODE REVIEW

File: client/integrationTest/nodeTests/integrationNode.test.ts [hunk 1]
```diff
@@ -203,7 +203,10 @@ describe('Integration Tests for Node Smart Contracts', () => {
                 await integrationTestUtil.generateSmartContractTests(smartContractName, '0.0.1', language, FabricRuntimeUtil.LOCAL_FABRIC);
                 // TODO: the generated smart contract tests for a generated smart contract can never pass out of the box
```
Comment (cazfletch) [inline client/integrationTest/nodeTests/integrationNode.test.ts]: can you update the comment as it now only applies to javascript

File: client/integrationTest/nodeTests/integrationNode.test.ts [hunk 2]
```diff
@@ -503,7 +506,11 @@ describe('Integration Tests for Node Smart Contracts', () => {
                 await integrationTestUtil.generateSmartContractTests(smartContractName, '0.0.1', language, 'myGateway');
                 // TODO: the generated smart contract tests for a generated smart contract can never pass out of the box
                 // until the tests are edited to provide the correct set of arguments for each transaction.
```
Comment (cazfletch) [inline client/integrationTest/nodeTests/integrationNode.test.ts]: again update the comment

File: client/integrationTest/nodeTests/integrationNode.test.ts (hunk 1 above)
Comment (Jakeeyturner) [reply client/integrationTest/nodeTests/integrationNode.test.ts]: Updated!

File: client/integrationTest/nodeTests/integrationNode.test.ts (hunk 2 above)
Comment (Jakeeyturner) [reply client/integrationTest/nodeTests/integrationNode.test.ts]: Updated!

**Pull request merged**